arm-image-installer

```diff
@@ -190,17 +190,21 @@ while [ $# -gt 0 ]; do
 				exit 1
 			fi
 			;;
-		--ign-url)
```

Comment:
I think if you leave this here it would allow the option to continue to work for those that may have scripted this